tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -606,7 +607,7 @@ def test_for_all_csv_compression_keywords(session, temp_schema, mode):
         session.sql(f"drop file format {format_name}")
 
 
-@pytest.mark.localtest
+# @pytest.mark.localtest
```

Comment:
temporarily skipped because the underlying table registration is reimplemented.
will flip in the following file operation get support